config/routes.rb

```diff
@@ -131,6 +131,14 @@
     end
   end
 
+  if defined?(Sufia::StatsAdmin)
+    namespace :admin do
+      constraints Sufia::StatsAdmin do
```

Comment:
@jcoyne As far as I could tell by reading the CanCanCan docs the ability is added based on a model.  We do not have a model defined for the stats.  Additionally this matches with the resque admin routes, so it is not a new pattern